### DISCUSSION TITLE
Position fixed div not working when scrolling

### DISCUSSION THREAD

**Discussion by kryptoDev0777** [community/community#40414] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am working on Typescript.
I tried to add fixed button on bottom and right.
but I can see that button on end of page, not fixed position.

**Comment by Preetiraj3697** [comment]
import { Container } from "@material-ui/core";
<Container fixed>
    <Button
    key="copy-to-clipboard"
    id="copy-to-clipboard"
    text="Copy"
    icon="file_copy"
    width={120}
    height={40}
    borderRadius={"3px"}
    onClick={copyToClipboard}
    margin="5px 10px"
     />
</Container>
